Cannot find module 'loopback-phase'

$ slc loopback
Error loading module "loopback":
 { [Error: Cannot find module 'loopback-phase'] code: 'MODULE_NOT_FOUND' }

Looks like we need to update the package.json with loopback-phase as dependency. 
Blocking

--- a/package.json
+++ b/package.json
@@ -36,18 +36,19 @@
     "bcryptjs": "~2.0.2",
     "body-parser": "~1.8.1",
     "canonical-json": "0.0.4",
+    "continuation-local-storage": "~3.1.1",
     "debug": "~2.0.0",
     "ejs": "~1.0.0",
     "express": "^4.10.2",
     "inflection": "~1.4.2",
     "loopback-connector-remote": "^1.0.1",
+    "loopback-phase": "^1.0.1",
     "nodemailer": "~1.3.0",
     "nodemailer-stub-transport": "~0.1.4",
     "strong-remoting": "^2.4.0",
     "uid2": "0.0.3",
     "underscore": "~1.7.0",
-    "underscore.string": "~2.3.3",
-    "continuation-local-storage": "~3.1.1"
+    "underscore.string": "~2.3.3"
   },
   "peerDependencies": {
     "loopback-datasource-juggler": "^2.8.0"
@@ -78,7 +79,6 @@
     "karma-script-launcher": "~0.1.0",
     "loopback-boot": "^1.1.0",
     "loopback-datasource-juggler": "^2.8.0",
-    "loopback-phase": "^1.0.1",
     "loopback-testing": "~0.2.0",
     "mocha": "~1.21.4",
     "serve-favicon": "~2.1.3",

